looprpc/client.proto

```diff
@@ -1440,3 +1446,24 @@ message InstantOut {
     */
     string sweep_tx_id = 5;
 }
+
+service StaticAddressClient {
+    /*
+    NewAddress requests a new static address for loop-ins from the server.
+    */
+    rpc NewAddress (NewAddressRequest) returns (NewAddressResponse);
+}
+
+message NewAddressRequest {
```

Comment:
This type is renamed later in this PR. I propose to squash this commit and the commit where it is renamed.